### PR TITLE
Initial suggestions for Group Norms

### PR DESCRIPTION
**Pull request by Awaab98**
This pull request closea #13 and introduces an **initial work frame** for our group's norms, compiled from the contributions made by all team members in the shared Google Docs group norm file.

---

## Key Points:
- This is a **summary and synthesis** of all group norms submitted so far.  
- The framework serves as a **starting point** for our discussion and collaboration.  
- It is **subject to review and modification**—team members are encouraged to:
  - **Add new norms** they believe should be included.  
  - **Modify existing norms** to better reflect group expectations.  
  - **Delete norms** that may not align with our collective goals.  

---

## Next Steps:
Please review the proposed framework and provide feedback by commenting on or suggesting changes in this pull request. Your input will help us finalize a set of norms that reflects the shared values and norms so that we can merge the pull request **during our next meeting**.

### CODE REVIEW

File: collaboration/README.md [hunk 1]
```diff
@@ -2,4 +2,55 @@
...
+and focused on progress, while conflicts are resolved respectfully. Flexibility
+and support ensure individual growth and team success.
+
+---
+
 <!-- group norms list -->
+
+## Group Norms  
+
+1. **Open Communication**
```
Comment (abeddost) [inline collaboration/README.md:18]: **Channels**: Use Slack for updates and general communication, and GitHub for technical discussions. Zoom or other tools may be used for synchronous discussions.
**Response Time**: Respond to messages within one working day on weekdays to ensure timely progress.

File: collaboration/README.md [hunk 2]
```diff
@@ -2,4 +2,55 @@
...
+   Share ideas freely, listen actively, and use platforms like Slack and GitHub
+   for updates and collaboration.
+
+2. **Respect for Time**  
+   Adhere to meeting schedules, start and end meetings punctually, and communicate
+   any delays or absences in advance.
+
+3. **Constructive Feedback**  
+   Give specific, respectful, and actionable feedback to help team members grow
+   and improve.
```
Comment (abeddost) [inline collaboration/README.md:28]: Use a feedback framework like "What went well, What can improve, and How to improve" for clarity and effectiveness.

File: collaboration/README.md [hunk 3]
```diff
@@ -2,4 +2,55 @@
...
+4. **Asynchronous Work**  
+   Update task progress regularly and respect individual schedules while ensuring
+   timely completion of tasks.
+
+5. **Team Support**  
+   Assist members facing challenges by sharing resources, guidance, and technical
+   expertise.
+
+6. **Conflict Resolution**  
+   Address disagreements calmly, focusing on ideas rather than individuals, and
```
Comment (abeddost) [inline collaboration/README.md:39]: Resolve disagreements by focusing on ideas rather than individuals.

File: collaboration/README.md [hunk 4]
```diff
@@ -2,4 +2,55 @@
...
+   Build trust through consistent communication, meet deadlines, and handle damaged
+   trust openly and maturely.
+
+8. **Inclusivity and Respect**  
+   Value diverse perspectives and ensure everyone has a chance to participate without
+   fear of exclusion or disrespect.
+
+9. **Documentation and Clarity**  
+   Maintain clear and updated documentation for tasks, code, and decisions to support
+   collaborative work.
```
Comment (abeddost) [inline collaboration/README.md:52]: Use templates for consistent documentation across the team.

**Review by abeddost — COMMENTED** [review]
Thank you for summarizing the group project norms! I’ve added a few suggestions to ensure they align with our discussions. Let me know your thoughts!

File: collaboration/README.md (hunk 1 above)
Comment (Awaab98) [reply collaboration/README.md:18]: Changes are reasonable and has been added.

**Review by abeddost — APPROVED** [review]
Hey @Awaab98, thanks for adding my suggestions. I reviewed everything, and it looks perfect. I'm merging the pull request.

**Comment by Yevheniia-Rudenko** [comment]
@Awaab98 @abeddost, thank you so much for creating the norms file. It looks really good, and I can fully support each norm! Great job!

**Comment by Allukmon** [comment]
wow this is really nice. great job!

**Pull request merged**